azure-mgmt-datafactory/azure/mgmt/datafactory/operations/integration_runtimes_operations.py

```diff
@@ -1041,3 +1041,69 @@ def upgrade(
             client_raw_response = ClientRawResponse(None, response)
             return client_raw_response
     upgrade.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/integrationRuntimes/{integrationRuntimeName}/upgrade'}
+
+    def remove_links(
+            self, resource_group_name, factory_name, integration_runtime_name, factory_name1, custom_headers=None, raw=False, **operation_config):
+        """Remove all linked integration runtimes under specific data factory in a
+        self-hosted integration runtime.
+
+        :param resource_group_name: The resource group name.
+        :type resource_group_name: str
+        :param factory_name: The factory name.
+        :type factory_name: str
+        :param integration_runtime_name: The integration runtime name.
+        :type integration_runtime_name: str
+        :param factory_name1: The data factory name.
+        :type factory_name1: str
+        :param dict custom_headers: headers that will be added to the request
+        :param bool raw: returns the direct response alongside the
+         deserialized response
+        :param operation_config: :ref:`Operation configuration
+         overrides<msrest:optionsforoperations>`.
+        :return: None or ClientRawResponse if raw=true
+        :rtype: None or ~msrest.pipeline.ClientRawResponse
+        :raises: :class:`CloudError<msrestazure.azure_exceptions.CloudError>`
+        """
+        linked_integration_runtime_request = models.LinkedIntegrationRuntimeRequest()
```

Comment:
@lmazuel - this appears to be an autorest generator bug: `LinkedIntegrationRuntimeRequest` takes `factory_name` as a required argument. In this case it seems after the parameter was renamed to `factory_name1` due to the conflict then it's no longer passed to the model.